dataactbroker/README.md

```diff
@@ -621,9 +622,15 @@ This route sends a request to the backend with the submission ID and file name t
 ```
 
 ##### Request Params
-  * file - a string indicating the file to generate (file name without extension)
-  * submission - the integer submission ID
-  
+  * warning - Whether or not the requested report is a warning (or error)
+    report. Defaults to False if this parameter isn't present.
+  * file\_type - One of 'appropriations', 'program\_activity',
+    'award\_financial', 'award', 'award\_procurement', 'awardee\_attributes'
+    or 'sub\_award'. Designates the type of report you're seeking.
+  * cross\_type - If present, indicates that we're looking for a
+    cross-validation report between `file_type` and this parameter. It accepts the
```

Comment:
I don't think you have to escape out the `_`, check and see? No need to add those extra `\` if you don't.